api/client/commands.go

```diff
@@ -392,6 +392,10 @@ func (cli *DockerCli) CmdLogin(args ...string) error {
 		return err
 	}
 	registry.SaveConfig(cli.configFile)
+
+	// encourage user to use `docker logout` in order to cleanup persisted login credentials
+	fmt.Fprintf(cli.out, "WARNING: login credentials persisted in %s If this is not a your personal machine, please `docker logout` to remove them when finished with the registry.\n", path.Join(os.Getenv("HOME"), registry.CONFIGFILE))
```

Comment:
No idea where they're at on this but there's a long discussion around the use of $HOME: https://github.com/docker/docker/pull/10602  In the meantime, I think you're supposed to use something like getHomeDir()